src/repository/mongoConnect.ts

```diff
@@ -0,0 +1,20 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { MongoClient, Db } from 'mongodb';
+import logger from '../logger';
+
+let db:Db;
+
+export async function connectToMongoDB() {
+  const uri = 'mongodb://localhost:27017';
```

Comment:
put these constants in a config and .env file (use dotenv).

You can do this in task 3